src/pydrex/visualisation.py

```diff
@@ -321,6 +464,11 @@ def figure_unless(ax):
     return fig, ax
 
 
+def figure():
+    """Create new figure with `dpi=300` and automatic constrained layout, grid etc."""
+    return plt.figure(dpi=300)
```

Comment:
Why not use `plt.rcParams["figure.dpi"] = 300`? Also, what do you mean by automatic constrained layout? I am not sure it is the default.

Reply:
Yeah I should put the DPI setting up the top in my rcParams stuff, like how I set up [constrained layout by default](https://github.com/seismic-anisotropy/PyDRex/blob/e0aa2c4b627cc26baf72872844d2be95d9a5d4f3/src/pydrex/visualisation.py#L18), at least for anything that imports `pydrex.visualisation`.